[terra-embedded-content-consumer] Enable content scrolling within the the frame (#1783)
cerner/terra-framework: packages/terra-embedded-content-consumer/CHANGELOG.md

## Unreleased

* Added

>   * Added scrolling prop and default to true to enable content scrolling within the iframe.

Reviewer: nit:
```suggestion
  * Added `scrolling` prop that is set to `true` by default to enable content scrolling within the iframe.
```

>   * Added an exmaple, and updated documentation on how to enable embedded content to be scrolled within the iframe.

Reviewer 2: ```suggestion
  * Added an example and updated documentation on how to enable embedded content to be scrolled within the iframe.
```
